src/faq/array/flattening-readonlyarray.ts

```diff
@@ -0,0 +1,7 @@
+import { pipe, Effect, ReadonlyArray } from "effect"
+
+// Effect.Effect<never, never, ReadonlyArray<number>>
```

Comment:
idem, should be `// $ExpectType Effect<never, never, number[]>`

Reply:
Done 👍